Zotero/Controllers/Sync/SyncScheduler.swift

```diff
@@ -70,6 +72,21 @@ final class SyncScheduler: SynchronizationScheduler, WebSocketScheduler {
     }
 
     var inProgress: BehaviorRelay<Bool>
+    var syncTypeInProgressObservable: Observable<SyncController.Kind?> {
+        return inProgress.map({ [weak self] progress in
```

Comment:
Why return a new observable instance for each access to this property, rather than e.g. initializing just one lazily?

Reply:
Fixed in d219e47b67c93d92933ce16a90215d5bb83168c3